.buildkite/pipeline.yaml

```diff
@@ -0,0 +1,11 @@
+steps:
+- command:
+  - "make lint"
+  label: ":llama: Lint"
+  agents:
+  - "queue=default"
+- command:
+  - "make test"
+  label: ":llama: Test"
+  agents:
+  - "queue=default"
```

Comment:
why not build?

Reply:
Skipped it as we are not using the bin anyways

Reply:
but we ensure that we can build it...

Reply:
Added the build step and also some more tests ;)